assets/js/base/components/chip/test/index.js

```diff
@@ -16,6 +16,14 @@ describe( 'Chip', () => {
 			expect( component.toJSON() ).toMatchSnapshot();
 		} );
 
+		test( 'should render nodes as the text', () => {
+			const component = TestRenderer.create(
+				<Chip text={ <h1>Test</h1> } />
+			);
+
+			expect( component.toJSON() ).toMatchSnapshot();
+		} );
```

Comment:
We should switch to use React Testing Library here. I understand though if you want to leave this for now and revisit in a potential follow-up.